src/mapwindow.jl

```diff
@@ -372,12 +422,6 @@ end
 # This is slightly faster than a circular buffer
 @inline cyclecache(b, x) = b[1], (Base.tail(b)..., x)
 
-replace_function(f) = f
-replace_function(::typeof(median!)) = function(v)
```

Comment:
What does `replace_function` do? Is it still needed? Tests still pass if I delete it.

Reply:
Okay, I saw that the replaced `median!` is faster, but wondered why a faster version lives here and not in `Base`. The comment about `keyword` arguments explains it. Can we get rid of this, as soon as `Base` has fast keywords?